### ISSUE TITLE
metar only shows after user with atis cycles tag

### ISSUE THREAD

**Issue by yahyaa506** [Arabian-vACC/OTDF-Doha-FIR-Controller-Pack#3]

**Comment by DedpewlControl** [comment]
Again, could I get a better description of the issue.

**Comment by DedpewlControl** [comment]
That is correct as the old version of Euroscope (version that everyone was was asked to update to due to the new authentication system) cannot distinguish between a split departure and arrival ATIS.

If you want to see the different ATIS Identifiers you'll need to update your Euroscope to the latest beta from euroscope.hu.

**Comment by Badger-Mush** [comment]
Closed due to needing a ES update.
